KataVista/utils/Description/Tips.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace BedrockStreamingUtils\CodingDojo\KataVista\Description;
+
+class Tips
+{
+    const HELPER_SIMPLE = 'Do not try to overthink things, you can always refactor later.';
+    const HELPER_MULTIPLE = 'You can use multiple helpers inside a single test to check all the new rule properties.';
+    const HELPER_POSITION = 'There is a helper function to check that a landmark is in a specific position (right/left).';
+    const HELPER_GROUND = 'There is a helper function to check that a landmark is in a specific plan (foreground/middleground/background).';
+    const HELPER_NUMBER = 'There is a helper function to check that a landmark is present multiple times.';
+    const HELPER_STACK = 'There is a helper function to check that a landmark is stacked over another one.';
+    const HELPER_RELATIVE = 'The new rule can help  position previously discovered elements. This might be the time for some refactoring in the Painter.';
+    const YOLO = 'At this point you should be able to got a good look at a coding dojo adn TDD. Good luck trying to fit this new rule ;)';
```

Comment:
`const YOLO = 'At this point you should be able to got a good look at a coding dojo and TDD. Good luck trying to fit this new rule ;)';`

Reply:
Fixed